### ISSUE TITLE
time zone specific date in response when requested

### ISSUE THREAD

**Issue by iameugenejo** [Edgio/hyperion#19]
Current spec requires any date time field in response to be in UTC timezone.

It's generally client's responsibility to format the date time field,

but, is there a use case that we want to allow users to ask for date time field in response in specific timezone?

**Comment by ardiesaeidi** [comment]
My concern with allowing this is now the API developer will have to manage datetime properly and that becomes a slippery slope.

- UTC is more reliable, the client can display the date and time in whatever format they want
- Users are in timezones, not offsets and that can change over time
  - What happens if a client saves the converted date values and they are no longer correct? 
  - For example, what if california no longer observes DST but a client stores those values?

**Comment by teehill** [comment]
So just use UTC with offset and avoid the daylight savings debacle?

**Comment by iameugenejo** [comment]
I agree, but what do we do when customers want the date fields in their timezone `offset` at the time of their request? 

Interpreting the time difference due to timezone change after DST change is on them in that case, of course.

**Comment by iameugenejo** [comment]
By the way, I have 0 problem with responding UTC date fields as long as I can include non-UTC date time for downloadable contents for this issue - https://github.com/VerizonDigital/hyperion/issues/20

**Comment by ardiesaeidi** [comment]
> So just use UTC with offset and avoid the daylight savings debacle?

No unfortunately offset is heavily dependent on the timezone and that is also affected by DST.
